### DISCUSSION TITLE
AI Prompt Template Using Syntax

### DISCUSSION THREAD

**Discussion by dorich** [chhoumann/quickadd#658] [Q&A] [answered]
I see an example in an early [video](https://www.youtube.com/watch?v=C4hCESoNo7k) where in addition to the returned response {{value}} there is a template item {{Selection}}. How does that function in the process of asking a question and getting a response? I tried creating a note with a question and selected the text of the question before invoking *Quick Add*  but that didn't seem to accomplish anything.
I'm interested to know how this can be used to increase productivity.

Apologies for the basic question but I'm not a programmer/software engineer.

Thanks

**Comment by chhoumann** [comment, marked as answer]
Hi @dorich!

`{{selected}}` will be either the text you've currently selected. If you haven't highlighted anything, it'll be empty - i.e., nothing.

Here's a video demonstration of how it works:

https://github.com/chhoumann/quickadd/assets/29108628/2fcb8d71-f255-4a6b-b0cf-f8572fd5ff4c

**Comment by dorich** [comment]
@chhoumann 
Thanks for your prompt answer.

Problem was cockpit error. I was not paying attention and typing *Selection* instead of *Selected*.
Thanks

**Reply by chhoumann** [reply]
It happens. Glad we got it solved :)
